rest/src/db/CatapultDb.js

```diff
@@ -374,6 +375,26 @@ class CatapultDb {
 		const sortConditions = { [sortingOptions[options.sortField]]: options.sortDirection };
 		const conditions = buildConditions();
 
+		// Check multisig graph if address is used in search criteria for cosigning,
+		// Then, show transactions for other cosigers.
+		if (filters.address !== undefined) {
+			return new MultisigDb(this).multisigsByAddresses([filters.address])
+				.then(multisigEntries => {
+					const addresses = [];
+					if (multisigEntries.length) {
+						if (multisigEntries[0].multisig.multisigAddresses.length)
+							addresses.push(...multisigEntries[0].multisig.multisigAddresses);
+					}
+					return addresses;
+				}).then(multisigAddresses => {
+					if (multisigAddresses && multisigAddresses.length) {
+						const buffers = multisigAddresses.map(address => address.buffer);
+						conditions['meta.addresses'] = { $in: buffers };
```

Comment:
This may be overriding the return { 'meta.addresses': Buffer.from(filters.address) }; above.

why not converting this function to async-await? So there is only one `this.queryPagedDocuments(` exit point